.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file never should have been committed. It would be a kindness if you could `git rm` it as part of this PR.

Reply:
Yeah, I `git rm --cached` this file. So it should not show up in our history after this.

Reply:
OK cool, my mistake.